src/GraphNodeManagerViewExtension/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -27,6 +30,24 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    internal class InverseBooleanToVisibilityConverter : IValueConverter
```

Comment:
Also the converters can be used from converters shared resource and keep the new ones here.

Reply:
Replaced all common converters.